datacube/testutils/io.py

```diff
@@ -391,16 +532,15 @@ def rio_slurp(fname, *args, **kw):
         return rio_slurp_read(fname, *args, **kw)
 
 
-def rio_slurp_xarray(fname, *args, rgb: str = "auto", **kw) -> xarray.DataArray:
```

Comment:
Is there a reason to have a separate set of low-level rasterio load paths here instead of using the rio driver in odc-loader?

Reply:
OK, we'll think about deprecating them in the next pass then.